src/pg_to_evalscript/javascript_processes/round.js

```diff
@@ -0,0 +1,26 @@
+function round(arguments) {
+  const { x, p = 0 } = arguments;
+
+  if (typeof x === undefined) {
+    throw new Error("Mandatory argument `x` is not defined.");
+  }
+
+  if (x === null) {
+    return null;
+  }
+
+  if (typeof x !== "number") {
+    throw new Error("Argument `x` is not a number.");
+  }
+
+  if (typeof p !== "number") {
```

Comment:
Since `p` is only allowed to be integer, the function should also check for that (`Number.isInteger(p)`) and return an error. Please also add a test for that.

Reply:
Added an additional check and test for this

Reply:
Thanks!

Reply:
Is this check still needed if we have `if (!Number.isInteger(p))` check?

Reply:
Yes I think its not needed. Removed

Reply:
Thanks, nicely spotted, went through it too quickly. Yeah, not needed